spec/view_helper/applinks_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe MetaTags::ViewHelper, 'displaying Applinks meta tags' do
+  subject { ActionView::Base.new }
+
+  it 'should display meta tags specified with :al' do
+    subject.set_meta_tags(al: {
+      foo:       'bar',
+    })
+    subject.display_meta_tags(site: 'someSite').tap do |meta|
+      expect(meta).to have_tag('meta', with: { content: "bar", property: "al:foo" })
+    end
+  end
+
+  it 'should use deep merge when displaying open graph meta tags' do
+    subject.set_meta_tags(al: { foo: 'bar' })
+    subject.display_meta_tags(al: { fooo: 'baar' }).tap do |meta|
+      expect(meta).to have_tag('meta', with: { content: "bar", property: "al:foo" })
+      expect(meta).to have_tag('meta', with: { content: "baar", property: "al:fooo" })
+    end
+  end
+
+  it "should not display meta tags without content" do
+    subject.set_meta_tags(al: {
+      title:       '',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.